### PR TITLE
[CP] buttons and rubocop-ing

### PR DESCRIPTION
**Pull request by CapCap**
Last minute changes for clarity

https://user-images.githubusercontent.com/1482859/168214595-d342b104-c97a-4012-9135-87aa13618f91.mov

and

image

### CODE REVIEW

File: ecosystem/platform/server/config/environments/development.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@
 
   # Suppress logger output for asset requests.
   config.assets.quiet = true
-
+  config.assets.debug = true
```
Comment (davidiw) [inline ecosystem/platform/server/config/environments/development.rb:78]: is this intentional?

**Review by davidiw — APPROVED** [review]

File: ecosystem/platform/server/app/views/shared/_captcha_helper.html.erb [hunk 2]
```diff
@@ -16,26 +16,4 @@
...
-        40% { transform: translate(1px, -1px) rotate(1deg); }
-        50% { transform: translate(-1px, 2px) rotate(-1deg); }
-        60% { transform: translate(-3px, 1px) rotate(0deg); }
-        70% { transform: translate(3px, 1px) rotate(-1deg); }
-        80% { transform: translate(-1px, -1px) rotate(1deg); }
-        90% { transform: translate(1px, 2px) rotate(0deg); }
-        100% { transform: translate(1px, -2px) rotate(-1deg); }
-    }
-</style>
+<%= render 'shared/shake_style' %>
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/views/shared/_captcha_helper.html.erb:19]: Just put it in application.css

File: ecosystem/platform/server/config/environments/development.rb (hunk 1 above)
Comment (CapCap) [reply ecosystem/platform/server/config/environments/development.rb:78]: yeah! that debug flag is on purpose: `/config/environments/development.rb` rails has different configs per env; the assets debug flag was the missing piece of why I had to kill my server every time I needed to recompile tailwind :-)

**Pull request merged**